src/Evolution/Systems/ScalarTensor/Sources/ScalarSource.hpp

```diff
@@ -0,0 +1,90 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "Evolution/Systems/ScalarTensor/Sources/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace ScalarTensor {
+
+/*!
+ * \brief Add in the source term to the \f$\Pi\f$
+ * evolved variable of the ::CurvedScalarWave system.
+ *
+ * \details The only source term in the wave equation
+ * \f[
+ *  \Box \Psi = \mathcal{S} ~,
+ *  \f]
+ *
+ * is in the equation for \f$\Pi\f$:
+ * \f[
+ *  \partial_t \Pi + \text{\{spatial derivative
+ * terms\}} = \alpha \mathcal{S}
+ * ~,
+ * \f]
+ *
+ * where \f$\mathcal{S}\f$ is the source term (e. g. in the Klein-Gordon
+ * equation, the source term is the derivative of the scalar potential
+ * \f$\mathcal{S} \equiv \partial V / \partial \Psi \f$.)
+ *
+ * (Note that this function takes as argument the source term.)
```

Comment:
Use `\param` to document each argument. Also in function below. Note if the return-by-reference argument is sized by the function.

Reply:
Added